### PR TITLE
Add function for converting State Abbreviation to/from FIPS

### LINKED ISSUE #137: Add fips to/from abbr to Python package
They are being added to R in #136, need to do the same for Python.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1198968030437421/1198970300808983) by [Unito](https://www.unito.io/learn-more)

### PR DESCRIPTION
**Pull request by chinandrew**
Closes #137 
Summary of changes:

* Add fips to abbr function and abbr to fips function, which requires doing basic string manipulation on the mapping file.
* update documentation

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1198968030437421/1198970435180742) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

File: Python-packages/covidcast-py/covidcast/geography.py [hunk 1]
```diff
@@ -128,6 +133,33 @@ def name_to_abbr(name: Union[str, Iterable],
...
 
 
+def fips_to_abbr(code: Union[str, Iterable],
+                 ignore_case: bool = False,
+                 fixed: bool = False,
+                 ties_method: str = "first") -> list:
+    """Look up state abbreviation by FIPS codes with regular expression support.
+
+    Given an individual or list of FIPS codes or regular expressions, look up the corresponding
+    state abbreviation
```
Comment (capnrefsmmat) [inline Python-packages/covidcast-py/covidcast/geography.py:143]:
```suggestion
    state abbreviation.
```

Comment (capnrefsmmat) [reply Python-packages/covidcast-py/covidcast/geography.py:143]: Do you want to give an example? Does this work with 2-digit or 5-digit FIPS?

**Review by capnrefsmmat — CHANGES_REQUESTED** [review]
Looks good. We should just probably be explicit about what a "state FIPS" means, since it could either be 2-digit or 5-digit

File: Python-packages/covidcast-py/covidcast/geography.py (hunk 1 above)
Comment (chinandrew) [reply Python-packages/covidcast-py/covidcast/geography.py:143]: Both, 
```
In [2]: covidcast.geography.fips_to_abbr("12")
Out[2]: ['FL']

In [3]: covidcast.geography.fips_to_abbr("12000")
Out[3]: ['FL']
```
I'll add a note in the docs

File: Python-packages/covidcast-py/covidcast/geography.py [hunk 2]
```diff
@@ -162,6 +195,34 @@ def name_to_cbsa(name: Union[str, Iterable],
...
 
+def abbr_to_fips(code: Union[str, Iterable],
+                 ignore_case: bool = False,
+                 fixed: bool = False,
+                 ties_method: str = "first") -> list:
+    """Look up state FIPS codes by abbreviation with regular expression support.
+
+    Given an individual or list of state abbreviations or regular expressions,
+    look up the corresponding state FIPS codes. The returned codes are 5 digits: the
+    2 digit state fips and 000 appeneded to the end.
```
Comment (capnrefsmmat) [inline Python-packages/covidcast-py/covidcast/geography.py:206]:
```suggestion
    2 digit state FIPS with 000 appended to the end.
```

Comment (chinandrew) [reply Python-packages/covidcast-py/covidcast/geography.py]: thanks

**Pull request merged**